### DISCUSSION TITLE
[✨] Add a "discovering Qwik Community plugins" capability for the Qwik CLI

### DISCUSSION THREAD

**Discussion by shairez** [QwikDev/qwik-evolution#108] [Proposals For Qwik]
### Is your feature request related to a problem?

We plan to move integrations into their own community plugins repos

So we need to write a capability for  the Qwik CLI to be able to discover installed community integration plugins and to use them in the CLI

### Describe the solution you'd like

1. A convention for npm packages like `qwik-plugin-x`
2. A plugin manifest file containing: "title", "description" and relevant files to install.

### Describe alternatives you've considered

n/a

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.

**Comment by shairez** [comment]
closing in favor of #20
